SU-SWS/stanford_media
# D8CORE-3967 Edited embeddable retains title, tests

# READY FOR REVIEW

# Summary
- Fix metadata method from erasing the media title.

# Need Review By (Date)
- 6/5

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. create an embeddable media entity with anything in the unstructured embed code field.
3. edit the same media entity and change any character in the unstructured code field
4. save and verify the name on the media entity didn't get erased.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
